0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -159,9 +128,9 @@ sub-protocols carried by this packet.
 |:--|:--|:--|
 | 1 | `Response` | Response |
 | 2 | `Error` | Response |
-| 3 | `Prepare` | Request |
-| 4 | `Fulfill` | Request |
-| 5 | `Reject` | Request |
+| 3 | (deprecated) |  |
```

Comment:
To me "deprecated" means "may be removed in the future". If you don't describe those messages at all anymore I would assume that new implementations don't support them at all, so something like "(removed)" or "(outdated)" may be better.